packages/turbo-types/src/types/config.ts

```diff
@@ -105,11 +105,13 @@ export interface RootSchema extends BaseSchema {
   globalPassThroughEnv?: null | Array<EnvWildcard>;
 
   /**
+   * Deprecated as of Turborepo 2.0.0. Consider using `globalDependencies` instead.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globaldependencies
+   *
```

Comment:
Instead of linking to `globalDependencies` within the TSDoc for `globalDotEnv` can we please remove this `Documentation: ` section?  That keeps it consistent and avoids any possible confusion about what the documentation is for (i.e. not this field).